packages/babel-parser/src/parser/statement.ts

```diff
@@ -2570,7 +2576,14 @@ export default abstract class StatementParser extends ExpressionParser {
       );
     }
 
-    if (this.match(tt._const) || this.match(tt._var) || this.isLet()) {
+    if (
+      this.match(tt._const) ||
+      this.match(tt._var) ||
+      this.isLet() ||
+      (this.hasPlugin("explicitResourceManagement") &&
+        ((this.isContextual(tt._using) && this.startsUsing()) ||
```

Comment:
Nit: could we move the `this.isContextual(tt._using)` inside `this.startsUsing()` (and maybe rename it to `isUsing`, similar to `isLet`)? Just because we are checking `this.isContextual(tt._using)` first every time we call `this.startsUsing()` anyway.